karma.conf.js

```diff
@@ -9,33 +13,40 @@ module.exports = function(config) {
     reporters: ['dots'],
     browsers: ['Chrome', 'PhantomJS'],
     preprocessors: {
-      'test/**/*.js': ['webpack']
+      'test/**/*.js': ['rollup']
     },
 
     // list of files / patterns to load in the browser
     files: [
-      'test/**/*.js'
+      { pattern: 'test/**/*.js', watched: false }
```

Comment:
What's the reasoning for `watched: false`? `gulp ci` watches, so won't this make it not restart when the tests change?

Reply:
it's a recommended configuration which I found here: https://www.npmjs.com/package/karma-rollup-preprocessor